infrastructure/kube/keep-prd/bcoin-service.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: bcoin
+  namespace: default
+  labels:
+    app: bitcoin-node
+    type: bcoin
+spec:
+  ports:
+  - port: 8332
+    targetPort: 8332
+    name: rpc
+  - port: 8333
+    targetPort: 8333
+    name: mainnet
+  selector:
+    app: bitcoin-node
+    type: bcoin
```

Comment:
The repetitiveness between name, type, labels, and selectors in these configs is slowly driving me insane. These all feel like opportunities for screwups down the line---updating A but not B, resulting in weird behavior that's hard to diagnose. Starting to lean towards some sort of hybrid where we use a generator to create the configs, and then commit the generated configs for good diffing in PRs...

Anyway, way outside the scope of this PR.

Reply:
I took setting up prd as an opportunity to revisit labels.  The idea is to match lesser environment deployments to these labels, so they're all the same.

Reply:
If you want to have a separate discussion about labeling patterns I'm happy to do that.

Reply:
I actually like that labels, names, and types are all related, I think it was a good update. What I'm saying instead is that this consistent style may be an opportunity to work to define something once and get a matching YAML, vs having to repeat it 3 times in 20 lines. I realize that can be a slippery slope though; definitely worth a separate discussion.

Reply:
Ah I see, with you there.  This config layer needs some attention for sure, and I'm not sure Helm is the answer.

I think part of me has intentionally kept this layer as dumb as possible until there's an opportunity to research a more elegant solution.